src/CommunityToolkit.Maui/Alerts/Snackbar/Snackbar.android.cs

```diff
@@ -179,13 +179,13 @@ public SnackbarCallback(in Snackbar snackbar, in TaskCompletionSource<bool> dism
 			this.dismissedTCS = dismissedTCS;
 		}
 
-		public override void OnShown(Object transientBottomBar)
+		public override void OnShown(Object? transientBottomBar)
```

Comment:
Let's use `object` instead of `Object`

```suggestion
		public override void OnShown(object? transientBottomBar)
```

Reply:
it is Java Object (line 8)